LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We probably want an even more permissive license since we want people to be copying this code without having to worry about licensing.

Reply:
changed to GNU General Public License v3.0